### PR TITLE
Release notes for 7.0.0

### LINKED ISSUE #2279: Write release notes for v7
# Interaction States and Attention Levels

TBD

### PR DESCRIPTION
**Pull request by jkaltoft**
## Which issue does this PR close?

This PR closes #2279 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: RELEASENOTES.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+### What's new?
+
+- [Button attention level 4 has been deprecated](#button-attention-level-4-has-been-deprecated)
+- [Interaction States](#interaction-states)
+- Charts
+  - New _Stock Chart_ component
+  - _Highcharts_ dependency removed
+  - General refactoring
+- Refactor moving of modal elements to solve multiple issues
```
Comment (RasmusKjeldgaard) [inline RELEASENOTES.md:19]: Could we be a bit more specific on what these multiple solved issues are?

**Review by RasmusKjeldgaard — COMMENTED** [review]
Reading through this, I was thinking that this style of communication might be better suited for an announcement in Github Discussions. 

This way we have this as a softer introduction and argumentation for new changes and then we only store the code-related notes (migration notes and changelog) near code. 

What do you think?

File: RELEASENOTES.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  <dd>When an element receives focus you can interact with it using a keyboard. Types of interaction could be e.g. activating a button, following a link, checking/unchecking checkboxes or selecting an option. Elements receive focus by using <code>TAB</code> on a keyboard to navigate to it. Not all elements will (or should) receive focus.</dd>
+</dl>
+
+#### What needs to be done?
+
+On a high level: Not much. You should check that the Kirby components you use look and behave as described in this section. You do not have to check every instance of Kirby components. A few samples should suffice.
+
+You _should_ test more thoroughly if you have applied your own styling and/or behavior. This could be either modifications made to existing interaction states or to make up for lacking ones. In either case you should aim for removing modifications and prefer to use the new baked-in interaction states.
+
+#### How desktop ready are Kirby components?
```
Comment (RasmusKjeldgaard) [inline RELEASENOTES.md:54]: I think this overview is quite overwhelming. Maybe we should limit it to the 'After' table, to serve as an easy reference for the components with changes.

File: RELEASENOTES.md (hunk 1 above)
Comment (jkaltoft) [reply RELEASENOTES.md:19]: Yes. I'm not sure how much detail the POs need, though.

File: RELEASENOTES.md (hunk 2 above)
Comment (jkaltoft) [reply RELEASENOTES.md:54]: I agree. It's not easy to compare before and after anyway, when presented in two different tables.

File: RELEASENOTES.md (hunk 1 above)
Comment (RasmusKjeldgaard) [reply RELEASENOTES.md:19]: Yeah you are right. Maybe we should just reference the PR or the issues solved?

Comment (jkaltoft) [reply RELEASENOTES.md]: Ok, I added the titles of the individual issues plus a link to the PR

**Comment by jkaltoft** [comment]
> Reading through this, I was thinking that this style of communication might be better suited for an announcement in Github Discussions.
> 
> This way we have this as a softer introduction and argumentation for new changes and then we only store the code-related notes (migration notes and changelog) near code.
> 
> What do you think?

That makes a lot of sense. As a bonus, it should get easier for other people than developers (e.g., product owners... 🤔😏) to write the "soft release notes". It would also be a good place for consumers to comment on the release (on a high level).

Ok, I suggest we:

1. Finish this PR - if you have no further comments we're done 😎
2. Copy the release note to a Github Discussions announcement. Maybe as a sort of pre-announcement of the v7 release (or announcement of the v7 pre-release), since we haven't actually released it yet
3. Close this PR without merging

**Review by RasmusKjeldgaard — APPROVED** [review]
> > Reading through this, I was thinking that this style of communication might be better suited for an announcement in Github Discussions.
> > This way we have this as a softer introduction and argumentation for new changes and then we only store the code-related notes (migration notes and changelog) near code.
> > What do you think?
> 
> That makes a lot of sense. As a bonus, it should get easier for other people than developers (e.g., product owners... 🤔😏) to write the "soft release notes". It would also be a good place for consumers to comment on the release (on a high level).
> 
> Ok, I suggest we:
> 
> 1. Finish this PR - if you have no further comments we're done 😎
> 2. Copy the release note to a Github Discussions announcement. Maybe as a sort of pre-announcement of the v7 release (or announcement of the v7 pre-release), since we haven't actually released it yet
> 3. Close this PR without merging

We're good to go!

**Comment by jkaltoft** [comment]
Moved to Github Discussions: https://github.com/kirbydesign/designsystem/discussions/2404

**Pull request closed without merging**